plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -3467,4 +3468,87 @@ public void testTrinoCacheInvalidatedOnCreateTable()
 
         assertUpdate("DROP TABLE " + tableName);
     }
+
+    @Test
+    public void testTimestampCoercionOnCreateTable()
```

Comment:
What is the relevance of this test?

The issue being fixed is about CTAS  not about INSERT

Reply:
You are right - i've tested on `master`

```
trino> create table delta.tiny.t1 (ts timestamp);
Query 20240227_121851_00001_5rsxn failed: Unsupported type: timestamp(3)
io.trino.spi.TrinoException: Unsupported type: timestamp(3)
	at io.trino.plugin.deltalake.transactionlog.DeltaLakeSchemaSupport.validatePrimitiveType(DeltaLakeSchemaSupport.java:419)
```